google_compute_engine/logger.py

```diff
@@ -32,6 +32,7 @@ def Logger(name, debug=False, facility=None):
   """
   logger = logging.getLogger(name)
   logger.handlers = []
+  logger.addHandler(logging.NullHandler())
```

Comment:
You can add the null handler import here:
https://github.com/GoogleCloudPlatform/compute-image-packages/blob/master/google_compute_engine/compat.py